apps/src/templates/Button.jsx

```diff
@@ -206,7 +206,9 @@ class Button extends React.Component {
               <FontAwesome icon="spinner" className="fa-spin" />
             </span>
           )}
-          <span className={moduleStyles.textSpan}>{!isPending && text}</span>
+          {text && (
```

Comment:
@levadadenys Buttons without text are very slightly off-center without this change. Is this safe to make?

Before:
<img width="129" alt="Screenshot 2024-04-17 at 10 40 59 AM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/625ca75f-248b-4ae1-bd6d-c9e7a743fdc7">

After:
<img width="123" alt="Screenshot 2024-04-17 at 10 40 28 AM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/2770260c-8889-43c5-add1-03403accba32">

Reply:
Oh, this is not a Design system button.
To use DesignSystem Button you need to use this one https://github.com/code-dot-org/code-dot-org/blob/2b4ae36a680597fb148bdb4085f65b766b1e4884/apps/src/componentLibrary/button/Button.tsx

Reply:
Thank you, Denys! I updated it to use the Design System button and changed the colors to black. Mark plans to suggest tweaks and these will probably undergo more iteration. 

<img width="768" alt="Screenshot 2024-04-18 at 4 53 22 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/9a6ee9e0-9471-41e2-a545-e693651360cb">